src/components/orders/AmountsDisplay/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import BigNumber from 'bignumber.js'
+
+import { formatSmart, TokenErc20 } from '@gnosis.pm/dex-js'
+
+import { Order } from 'api/operator'
+
+import { HIGH_PRECISION_DECIMALS, HIGH_PRECISION_SMALL_LIMIT } from 'apps/explorer/const'
+
+import { BlockExplorerLink } from 'apps/explorer/components/common/BlockExplorerLink'
+
+import { RowContents, RowTitle, StyledImg, UsdAmount, Wrapper } from './styled'
+
+type RowProps = {
+  title: string
+  showTitleSuffix: boolean
```

Comment:
nit: This is for the `at least`, I would just call it `amountIsLowerBound` or sth in those lines. Or do sth like `amountIsFixed`, or `amountIsAtMost`, or `amountIsAtLeast`

if you want to call that title suffix, I would expect it to be an optional suffix string. But I think,

Reply:
You cut off in the middle, but I get the point.
Forgot that it's `at most` when selling and  `at least` when buying.
Think I'll instead take the suffix as a parameter instead of a boolean.
New PR coming soon

Reply:
https://github.com/gnosis/gp-ui/pull/209